sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -716,7 +713,8 @@ public ServiceBusSessionReceiverClientBuilder topicName(String topicName) {
         public ServiceBusReceiverAsyncClient buildAsyncClient() {
             final MessagingEntityType entityType = validateEntityPaths(logger, connectionStringEntityName, topicName,
                 queueName);
-            final String entityPath = getEntityPath(logger, entityType, queueName, topicName, subscriptionName);
+            final String entityPath = getEntityPath(logger, entityType, queueName, topicName, subscriptionName,
+                SubQueue.NONE);
```

Comment:
Why none? Don't we have some parameter user's set?

Reply:
Any sub queue are accessed via non-session receiver, confirmed with dotnet also. 
Thus we are not exposing `subQueue()` in session receiver builder. That is why I am passing NONE.